3ds/libsidplayfp/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: Sebastian Weber <me@badda.de>
+
+pkgname=3ds-libsidplayfp
+pkgver=1.8.8
```

Comment:
change this to
```
_basever=1.8
_patchver=8
pkgver=${_basever}.${_patchver}
```
which will make the url change easier

Reply:
ok